packages/remix-dev/compiler/plugins/serverBareModulesPlugin.ts

```diff
@@ -82,11 +82,16 @@ export function serverBareModulesPlugin(
         }
 
         switch (remixConfig.serverBuildTarget) {
-          // Always bundle everything for cloudflare.
+          // Always bundle everything for Cloudflare
           case "cloudflare-pages":
           case "cloudflare-workers":
           case "deno":
             return undefined;
+          case "netlify-edge":
+            // Bundle everything except URL imports
+            if (!path.startsWith("https:") && !path.startsWith("file:")) {
+              return undefined;
+            }
```

Comment:
The [urlImportsPlugin](https://github.com/remix-run/remix/blob/main/packages/remix-dev/compiler/plugins/urlImportsPlugin.ts) marks all URL imports as external, so they are never bundled, though we only check for `/https?:\/\//`, so we can probably simplify to:

```suggestion
          case "netlify-edge":
            // Do not bundle file URIs
            if (!path.startsWith("file:")) {
              return undefined;
            }
```